dll/cpl/input/input_list.c

```diff
@@ -323,14 +323,56 @@ InputList_AddInputMethodToUserRegistry(DWORD dwIndex, INPUT_LIST_NODE *pNode)
         }
     }
 
+    /* 
+     * First time more then 1 input language exists switching hotkeys needed
+     * set default values for language/layout switching hotkeys
```

Comment:
What do you exactly mean in this comment?

Reply:
Adjusted hopefully it is clear now